### PR TITLE
Um packing

### PR DESCRIPTION
**Pull request by pp-mo**
Provide wgdos packing for um output.
Replaces https://github.com/pp-mo/iris/pull/15
(as re-targetted from pp-mo/um_packed_passthru to SciTools/master)

### CODE REVIEW

File: lib/iris/tests/integration/test_FieldsFileVariant.py [hunk 1]
```diff
@@ -341,9 +341,9 @@ def test_save_packed_as_unpacked(self):
                          if hasattr(fld, 'lbpack')]
             for i_field, (new_size, old_size) in enumerate(zip(new_sizes,
                                                                old_sizes)):
-                msg = 'unpacked LBNREC({}) is <= packed({})'
-                self.assertGreater(new_size, old_size,
-                                   msg=msg.format(new_size, old_size))
+                msg = 'unpacked LBNREC({}) is < packed({})'
```
Comment (pp-mo) [inline lib/iris/tests/integration/test_FieldsFileVariant.py]: From older PR : https://github.com/pp-mo/iris/pull/15#discussion_r29036594
> @pelson Quoi? I don't understand... could you provide some context.

Attempt to check that packed payload is smaller, as it should be.
This is admittedly weak.  I will try using LBLREC instead, in which case it should be >, not >= ...

**Comment by pp-mo** [comment]
Older comments :
 * https://github.com/pp-mo/iris/pull/15#discussion_r29036450
   now fixed.  Prompted refactor of the code.
 * https://github.com/pp-mo/iris/pull/15#discussion_r29036594
   copied over, at: https://github.com/SciTools/iris/pull/1650#discussion_r29104733
   (still to address : watch this space...)

**Comment by rhattersley** [comment]
@pp-mo - I guess this is made obsolete by your upcoming "mule" work? Is there anything that needs to be salvaged?

**Comment by pp-mo** [comment]
> made obsolete by your upcoming "mule" work

Guess so.

**Pull request closed without merging**